### PR TITLE
Switch to Python 3

### PR DESCRIPTION
**Pull request by pinotree**
Time to switch unconditionally to Python 3, as Python 2 is EOL by the beginning of 2020.

This includes a mandatory fix to avoid generating broken C sources.

### CODE REVIEW

**Comment by OlafvdSpek** [comment]
Thanks!

File: src/htmlparser/fsm_config.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```
Comment (OlafvdSpek) [inline src/htmlparser/fsm_config.py]: Why was this removed?

Comment (pinotree) [reply src/htmlparser/fsm_config.py]: See the message of the commit that removes it.

In short: it is not a Python script to run (only a module to import), so it does not need it.
